openverse_catalog/dags/providers/provider_dag_factory.py

```diff
@@ -316,6 +316,9 @@ def create_provider_api_workflow(
                 "provider_name": provider_name,
                 "duration": XCOM_PULL_TEMPLATE.format(pull_data.task_id, "duration"),
                 "record_counts_by_media_type": record_counts_by_media_type,
+                "dated": dated,
+                "date_range_start": "{{ data_interval_start | ds }}",
+                "date_range_end": "{{ data_interval_end | ds }}",
```

Comment:
TIL, so nice that it's so easy to get the interval start/end!

Reply:
Airflow's template & macros are _so_ powerful!